plugin/trino-memory/src/test/java/io/trino/plugin/memory/TestMemorySmoke.java

```diff
@@ -437,24 +425,6 @@ private Session withPartitionedJoin()
                 .collect(toImmutableList());
     }
 
-    @Test
```

Comment:
moving of method should be separate commit so it's easier to review (now diff is scattered)